Tests/PointFreeTests/__Snapshots__/NewslettersTests/testExpressUnsubscribeReply_IncorrectSignature.1.Conn.txt

```diff
@@ -3,1999 +3,10 @@ Cookie: pf_session={}
 
 recipient=unsub-f9c46e50cb32c3f12369e92c8bb9d9db09edf2cce5a0307b4e8516ac36340b470d3fed478e73515f3ad396db9cb71d21c0abe5f9872e3e792de15f4b1ae5795e%40pointfree.co&sender=hello%40pointfree.co&signature=this%20is%20an%20invalid%20signature&timestamp=1517356800&token=deadbeef
 
-404 Not Found
-Content-Length: 45057
-Content-Type: text/html; charset=utf-8
+406 Not Acceptable
```

Comment:
for posterity this is a request that was previously 404'ing because of failed decryption in the router, but not the request gets to the middleware and we `406` it when decryption fails.